js/index.js

```diff
@@ -3,5 +3,108 @@ window.onload = () => {
     startGame();
   };
 
-  function startGame() {}
+  function startGame() {
+    // Declaramos la variables 
+    const canvas = document.getElementById("canvas");
+    const ctx = canvas.getContext("2d");
+
+    // Primera iteración
+    const roadImg = new Image();
+    roadImg.src = "./images/road.png"
+    roadImg.onload = function(){
+      ctx.drawImage(roadImg, 0, 0, canvas.width, canvas.height);  
+    }
+  
+    // Segunda iteración
+    const carImg = new Image();
+    carImg.src = "./images/car.png";
+    let carX = canvas.width / 2 - 25;
+    let carY = canvas.height - 100;
+    carImg.onload = function(){
+      ctx.drawImage(carImg,carX,carY,50,100);
+    };
+
+    // Tercera Iteración
```

Comment:
in this case define comments on english or all in spanish but no to mixed comments